### DISCUSSION TITLE
Shortcut or top level item for Milestones

### DISCUSSION THREAD

**Discussion by dotherightthing** [community/community#10271] [Mobile] [answered]
Can you please add a way to quickly jump to one milestone or all milestones in a project

**Comment by mishmanners** [comment]
This would be awesome.

**Comment by brianlovin** [comment, marked as answer]
This should be possible today. From the Home tab, edit your shortcuts → create a new shortcut → scope it to a specific repository → apply the milestone filter.

Let me know if that works!

**Reply by dotherightthing** [reply]
Thanks for the instructions, I didn't realise I could add a shortcut beyond what was on the 'Suggested shortcuts' list. However I'm still limited to choosing one Milestone per shortcut. Could the filters be updated to support multi select so I could choose several?

**Reply by brianlovin** [reply]
Interesting, yeah I can share this feedback with the team. Just curious: could you share more about your use case for wanting to filter my multiple milestones at one time? That context might help us understand how people use the feature better, so we can approach future shortcuts improvements better.

**Reply by dotherightthing** [reply]
Most of my projects are solo rather than team projects. I prefer Milestones over Labels as Milestones give me a completion meter. I like to be able to see what's due for the current milestone and the ones following. This helps me quickly prioritise sprints and tasks when talking to a client, and provides a visual roadmap of what's to come. So maybe I don't really want a giant pot of all milestones, but links to the next N milestones, without having to create each of these manually.

**Reply by brianlovin** [reply]
Gotcha. Are you using the latest Issues product yet? http://github.com/features/issues

**Comment by dotherightthing** [comment]
No, I wasn't aware of the issues product, thanks for the link. I just had a
quick play with it and upvoted some feedback around the bulk importing of
issues.
